shared/macros.jinja

```diff
@@ -379,9 +379,7 @@ ocil_clause: "the package is installed"
 
 {{%- macro systemd_describe_socket_disable(socket) %}}
     The <code>{{{ socket }}}</code> socket can be disabled with the following command:
-    <pre>$ sudo systemctl disable {{{ socket }}}.socket</pre>
-    The <code>{{{ socket }}}</code> socket can be masked with the following command:
-    <pre>$ sudo systemctl mask {{{ service }}}.socket</pre>
+    <pre>$ sudo systemctl mask {{{ socket }}}.socket</pre>
```

Comment:
To mask, we either need to disable the service first, or add `--now`. Otherwise, service is still running.

Reply:
Yes, we have to add `--now`, but `disable` has the same behavior, so the instruction wasn't accurate before either:

From the `systemctl` manpage
```
       disable UNIT...
           Disables one or more units. ...

           This command implicitly reloads the system manager configuration after completing the operation. Note that this command does not implicitly stop the units that are being disabled. If this is desired, either combine this
           command with the --now switch, or invoke the stop command with appropriate arguments later.
```